docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md

```diff
@@ -68,46 +68,53 @@ Not applicable as this is the initial release of GDC 2.0.
 
 ### Known Issues and Workarounds
 
-* Section 508 Accessibility:
+* __Section 508 Accessibility__:
     * There are known Section 508 accessibility issues that the GDC plans to address in subsequent releases. If a user encounters a Section 508 barrier, please contact GDC Support (support@nci-gdc.datacommons.io) for assistance. Known Section 508 issues are identified below.
-        * Heatmaps within the Sequence Reads tool do not contain concise alternative text or equivalent alternatives. Additionally, equivalent alternatives to the Box plots, QQ plots, Venn diagrams, and the body plot are not available.
         * There are keyboard focus and navigation issues in analysis tools that use popup windows/overlays for custom user selections. Impacted analysis tools include BAM Slicing, Sequence Reads, Gene Expression Clustering, OncoMatrix, and ProteinPaint.
+        * Heatmaps within the Sequence Reads tool do not contain concise alternative text or equivalent alternatives. Additionally, equivalent alternatives to the Box plots, QQ plots, Venn diagrams, and the body plot are not available.
```

Comment:
I may just be missing something, but what is the body plot?

Reply:
@clsackett it's the...giant body image on the home page. Maybe I can add "on the home page" to clarify -- I'll do that!

Reply:
Thanks for the clarification @wteouchicago ! Since the list contained elements that were part of Analysis tools, I was trying to think of a body plot there, but the edit helps clear it up